serverManager/common/include/ISessionServerAppManager.h

```diff
@@ -38,13 +38,15 @@ class ISessionServerAppManager
     ISessionServerAppManager &operator=(const ISessionServerAppManager &) = delete;
     ISessionServerAppManager &operator=(ISessionServerAppManager &&) = delete;
 
-    virtual bool initiateApplication(const std::string &appId, const firebolt::rialto::common::SessionServerState &state,
+    virtual void preloadSessionServers(int numOfPreloadedServers) = 0;
+    virtual bool initiateApplication(const std::string &appName,
+                                     const firebolt::rialto::common::SessionServerState &state,
                                      const firebolt::rialto::common::AppConfig &appConfig) = 0;
-    virtual bool setSessionServerState(const std::string &appId,
+    virtual bool setSessionServerState(const std::string &appName,
                                        const firebolt::rialto::common::SessionServerState &newState) = 0;
-    virtual void onSessionServerStateChanged(const std::string &appId,
+    virtual void onSessionServerStateChanged(int appName,
                                              const firebolt::rialto::common::SessionServerState &newState) = 0;
-    virtual std::string getAppConnectionInfo(const std::string &appId) const = 0;
+    virtual std::string getAppConnectionInfo(const std::string &appName) const = 0;
```

Comment:
I understand why you want to use an integer app ID internally. I just wonder if it might get confusing when externally we have a string appId, which internally we convert to a string appName with associated int appId. Can we think of some better terminology to avoid this?

Reply:
Adam suggested to substitute appId with serverId, I changed it and I think, that the terminology is not confusing now. We've got serverId for each running rialto server and appName for an app, which is associated with rialto server. Let me know, if it's still confusing for You :)